### DISCUSSION TITLE
List of pip commands throughout the course

### DISCUSSION THREAD

**Discussion by NewCoderrrrr** [smartcontractkit/full-blockchain-solidity-course-py#1322] [General]
May be missing some but these were the commands I used throughout the course (May change pip3 to pip):

pip3 install pip
pip3 install py-solc-x
pip3 install black
pip3 install web3
pip3 install python-dotenv
npm install ganache --global
ganache —deterministic 
pip3 install Cython
python3 -m pip install --user pipx
python3 -m pipx ensurepath (Close terminal and re-open)
pip3 install eth-brownie (Close terminal and re-open)
pip3 install pytest

**Comment by cromewar** [comment]
Yes there are more or less the commands you'll use on the course.
